sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -46,7 +47,7 @@ static bool IsSuitableSubReq(const Requirement *Req) {
 }
 
 Scheduler::GraphBuilder::GraphBuilder() {
-  if (const char *EnvVarCStr = std::getenv("SYCL_PRINT_EXECUTION_GRAPH")) {
+  if (const char *EnvVarCStr = SYCLConfig<SYCL_PRINT_EXECUTION_GRAPH>::get()) {
```

Comment:
Can we use it for all environment variable (SYCL_DEVICE_TYPE, SYCL_DUMP_IMAGES, etc)?

Reply:
We can use it for environment variables that are "getenv'ed" inside source directory.

Reply:
I suggest to mention this limitation in the commit message.

Reply:
Ok, will do during merge.

Reply:
updated description with new sentence.